packages/components/src/components/file-uploader/_file-uploader.scss

```diff
@@ -120,19 +173,58 @@
   }
 
   .#{$prefix}--file__state-container .#{$prefix}--file-complete {
-    fill: $support-02;
+    fill: $interactive-04;
     cursor: pointer;
 
     &:focus {
       @include focus-outline('border');
     }
+
+    // for checkmark contrast
+    [data-icon-path='inner-path'] {
+      opacity: 1;
+      fill: $ibm-color__white-0;
```

Comment:
Do we wanna use a theme color token for this rather than the ibm color tokens?

Reply:
which token are you suggesting?